Model/Config/Backend/Schedule.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Lfi\MonologDatabase\Model\Config\Backend;
+
+use Magento\Cron\Model\Config\Source\Frequency;
+use Magento\Framework\Exception\CouldNotSaveException;
+
+class Schedule extends \Magento\Framework\App\Config\Value
+{
+    const JOB_NAME = 'lfi_monolog_database_clean';
+
+    protected $configValueFactory;
+    protected $modelPath;
+
+    
+    public function __construct(
+        \Magento\Framework\Model\Context $context,
+        \Magento\Framework\Registry $registry,
+        \Magento\Framework\App\Config\ScopeConfigInterface $config,
+        \Magento\Framework\App\Config\ValueFactory $configValueFactory,
+        \Magento\Framework\App\Cache\TypeListInterface $cacheTypeList,
+        \Magento\Framework\Model\ResourceModel\AbstractResource $resource = null,
+        \Magento\Framework\Data\Collection\AbstractDb $resourceCollection = null,
+        $runModelPath = '',
+        array $data = []
+    ) {
+        parent::__construct($context, $registry, $config, $cacheTypeList, $resource, $resourceCollection, $data);
+        $this->configValueFactory = $configValueFactory;
+        $this->modelPath = $runModelPath;
+    }
+
+    /**
+     * {@inheritDoc}
+     * @throws CouldNotSaveException
+     */
+    public function afterSave()
+    {
+        $time = $this->getData('groups/monolog_database/fields/time/value');
+        $frequency = $this->getData('groups/monolog_database/fields/frequency/value');
+
+        $cronExprArray = [
+            (int)$time[1], // min
+            (int)$time[0], // hour
```

Comment:
I don't see the system.xml configuration of `time` requiring a value, and there aren't any defaults. How does this behave if time is left unset?

Reply:
Since it is a `type="time"` input you automatically get `00,00,00` when saving the config value.

Reply:
Nice, so it's impossible for the this input to not have array indices 0 and 1 at this point.